src/_sass/pages/_learning-resources-index.scss

```diff
@@ -0,0 +1,205 @@
+#resource-index-content {
+  display: flex;
+  flex-direction: row;
+
+  .left-col {
+    margin-right: 1rem;
+    flex: 2;
+  }
+
+  .right-col {
+    width: 220px;
+
+    @media (max-width: 840px) {
+      position: fixed;
+      top: var(--site-header-height);
+      bottom: 0;
+      right: -10rem;
+      background-color: var(--site-switcher-bg, #ffffff);
+      box-shadow: 0 6px 18px 0 rgba(0, 0, 0, 0.2);
+      border-radius: 0.4rem;
+      width: max-content;
```

Comment:
This causes resizing when expanding the collapsible lists. I'd consider choosing a width and wrapping entries if they don't fit.